app.py

```diff
@@ -1,24 +1,14 @@
 # -*- coding: utf-8 -*-
-import flask
-from pythainlp.tokenize import word_tokenize
-app = flask.Flask(__name__)
+from fastapi import Depends, FastAPI, Header, HTTPException
+from routers import tag, tokenize
 
-@app.route("/")
+app = FastAPI()
+
+
+@app.get("/")
 def hello():
-    return "Hello World!"
+    return {"Hello": "World"}
```

Comment:
It would be good if we return PyThaiNLP's version to this route, e.g. one can use it to for testing purposes.

Reply:
> 
> 
> It would be good if we return PyThaiNLP's version to this route, e.g. one can use it to for testing purposes.

OK. I fixed done.